atlassian_jwt_auth/contrib/server/helpers.py

```diff
@@ -32,25 +31,31 @@ def _requires_asap(verifier, auth, parse_jwt_func, build_response_func,
         if verify_issuers_func is not None:
             verify_issuers_func(asap_claims, issuers)
         asap_claim_holder.asap_claims = asap_claims
-    except requests.exceptions.HTTPError as e:
+    except PublicKeyRetrieverException as e:
+        if e.status_code != 404:
```

Comment:
@grantisu actually this is tricky because amazon s3 can reply with `403` when a file is missing.

Reply:
Fair enough.